projects/packages/forms/src/blocks/contact-form/components/jetpack-field-width.js

```diff
@@ -11,21 +15,25 @@ export default function JetpackFieldWidth( { setAttributes, width } ) {
 			className="jetpack-field-label__width"
 			__nextHasNoMarginBottom={ true }
 		>
-			<BaseControl.VisualLabel>{ __( 'Field Width', 'jetpack-forms' ) }</BaseControl.VisualLabel>
-			<ButtonGroup aria-label={ __( 'Field Width', 'jetpack-forms' ) }>
+			<ToggleGroupControl
+				__next40pxDefaultSize
+				__nextHasNoMarginBottom
+				aria-label={ __( 'Field Width', 'jetpack-forms' ) }
+				isBlock
+				label={ __( 'Field Width', 'jetpack-forms' ) }
+				onClick={ value => setAttributes( { width: value } ) }
```

Comment:
> I don't understand how the selected value is correctly white first, but when I change it, it stays grey

Untested, but maybe because of this?

```suggestion
				onChange={ value => setAttributes( { width: value } ) }
```

Reply:
Ahh of course, thanks :-)